dask/dataframe/core.py

```diff
@@ -1855,21 +1855,43 @@ def astype(self, dtype):
         empty = self._pd.astype(dtype)
         return map_partitions(pd.DataFrame.astype, empty, self, dtype=dtype)
 
-    def info(self):
+    def info(self, buf=None, verbose=False, memory_usage=False):
```

Comment:
So it looks like, by default, this doesn't require computation.  I'm generally in favor of this.